rotkehlchen/chain/evm/decoding/thegraph/balances.py

```diff
@@ -40,6 +46,78 @@ def __init__(
         self.token = native_asset.resolve_to_evm_token()
         self.staking_contract = staking_contract
 
+    def _query_vesting_balances(self, balances: 'BalancesSheetType') -> 'BalancesSheetType':
+        """Query balances of vested GRT tokens if DelegationTransfer events are found.
+        This function modifies and returns the 'balances'"""
+        db_filter = EvmEventFilterQuery.make(
+            counterparties=[self.counterparty],
+            location=Location.ETHEREUM,
+            event_types=[HistoryEventType.INFORMATIONAL],
+            event_subtypes=[HistoryEventSubType.NONE],
+        )
+        with self.event_db.db.conn.read_ctx() as cursor:
+            events = self.event_db.get_history_events(
+                cursor=cursor,
+                filter_query=db_filter,
+                has_premium=True,
+            )
+        if len(events) == 0:
+            return balances
+
+        vesting_contracts = defaultdict(list)
+        for event in events:
+            if (
+                event.location_label is None or
+                event.extra_data is None or
+                (delegator := event.extra_data.get('delegator_l2')) is None
+            ):
+                continue
+            vesting_contracts[string_to_evm_address(event.location_label)].append(
```

Comment:
when you skip something because data is incorrect (this comes from decoded events) always log something relevant to find the event and that you are skipping it